src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -105,6 +106,11 @@ define(function (require, exports, module) {
     function start(){
         window.addEventListener("message", _listener);
 
+        var autoUpdate = BrambleStartupState.ui("autoUpdate");
+        if(typeof autoUpdate === "boolean") {
```

Comment:
Probably just type check? I think for all the other properties, we don't do any coercion so maybe keep it like that for now?